### DISCUSSION TITLE
[Request] Arch Linux demo

### DISCUSSION THREAD

**Discussion by GrabbenD** [bootc-dev/bootc#142] [Q&A] [answered]
Is it actually possible to use distros which aren't based on RPM in the current state of this project?

**Comment by cgwalters** [comment]
I think the only sticking point right now is bootupd which is there for silly reasons.  The right place to ask about that is https://github.com/coreos/bootupd/issues/468

Will work on fixing that at some point.

**Comment by BoukeHaarsma23** [comment]
Hey, Sorry to kick this again. But I'm also very much interested in this. Is there any way for me to help?

**Reply by cgwalters** [reply]
Thanks for pinging on this. I would be happy to assist if someone is interested in digging into the code!

**Comment by GrabbenD** [comment, marked as answer]
**Future Readers**

This topic has been moved to here: https://github.com/bootc-dev/bootc/issues/865
